common.js

```diff
@@ -151,15 +145,13 @@ module.exports = {
 
   moveApp: function finalizeApp (opts, tempPath, callback) {
     var finalPath = generateFinalPath(opts)
-    // Prefer ncp over mv (which seems to cause issues on Win8)
-    series([
-      function (cb) {
-        mkdirp(finalPath, cb)
-      },
-      function (cb) {
-        ncp(tempPath, finalPath, cb)
-      }
-    ], function (err) {
+
+    if (opts.tmpdir === false) {
+      callback(null, finalPath)
+      return
+    }
+
+    fs.move(tempPath, finalPath, function (err) {
```

Comment:
fs-extra handles `EPERM` on windows [correctly](https://github.com/jprichardson/node-fs-extra/blob/master/lib/move/index.js). We don't need to use mkdirp/ncp here explicitly.